_episodes_rmd/08-plot-ggplot2.Rmd

```diff
@@ -194,7 +194,7 @@ lines.
 > >
 > > The lines now get drawn over the points!
 > >
-> > ```{r ch3-sol}
+> > ```{r ch3-sol, fig. alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The plot illustrates the possibilities for styling visualisations in ggplot2 with data points enlarged, coloured orange, and displayed without transparency."}
```

Comment:
```suggestion
> > ```{r ch3-sol, fig.alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The plot illustrates the possibilities for styling visualisations in ggplot2 with data points enlarged, coloured orange, and displayed without transparency."}
```